README.md

```diff
@@ -128,8 +128,51 @@ Docker
 
 - [Developing minetestserver with Docker](doc/developing/docker.md)
 
-We provide a Dockerfile that can be used to build the server.
+We provide Minetest server Docker images using the GitHub container registry.
 
+Images are built on each commit and available using the following tag scheme:
+
+* `ghcr.io/minetest/minetest:master` (latest build)
+* `ghcr.io/minetest/minetest:<tag>` (specific tag)
+* `ghcr.io/minetest/minetest:latest` (latest tag)
+
+See [here](https://github.com/minetest/minetest/pkgs/container/minetest) for the available tags.
+
+If you want to test it on a Docker server you can easily run:
+
+```shell
+sudo docker run ghcr.io/minetest/minetest:master
```

Comment:
root is not generally needed for docker, depends on setup. so we shouldn't reproduce it in our instructions